cv32e40s/tests/programs/custom/pushpop_debug_triggers/pushpop_debug_triggers.c

```diff
@@ -122,7 +123,7 @@ static void  setup_triggers(void){
   mcontrol6.fields.match = 0;  // (match exact address)
   mcontrol6.fields.type  = 6;
 
-  trigger_addr = (uint32_t) &(g_pushpop_area[2]);  // (arbitrary index)
+  trigger_addr = (uint32_t) &(g_pushpop_area[index]);  // (arbitrary index)
```

Comment:
`// (arbitrary index)`

This is no longer true.

But I don't really mind leaving it there, unless you already are about to change something else in the source.